lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -435,6 +456,22 @@ void StagesSubject::stageChanged(
             }
         }
 #endif
+
+        // Warn if a locked attribute value has been changed, probably through
+        // the USD API.
+        if (changedPath.IsPropertyPath()) {
```

Comment:
I had the same thought this morning :) Since Will confirmed that we will not support the case where the user would edit the attribute directly via the USD Python module, this check is warning about something we don't intend to support anyway. I think indeed it should be removed.